storage/s3_test.go

```diff
@@ -283,6 +283,10 @@ func TestS3Retry(t *testing.T) {
 			name: "RequestError",
 			err:  awserr.New(request.ErrCodeRequestError, "request error", nil),
 		},
+		{
+			name: "RequestError",
+			err:  awserr.New(request.ErrCodeRequestError, "use of closed network connection", nil),
```

Comment:
could you use a unique test name please?

Reply:
updated